### PR TITLE
Add stories for the content type switcher

### PR DESCRIPTION
**Pull request by dhruvkb**
## Description

This PR adds a story for the `VContentSwitcher` component. This should make it easier to develop #639 in isolation.

## Checklist

- [x] My pull request has a descriptive title (not a vague title like `Update index.md`).
- [x] My pull request targets the *default* branch of the repository (`main`) or a parent feature branch.
- [x] My commit messages follow [best practices][best_practices].
- [x] My code follows the established code style of the repository.
- [ ] I added or updated tests for the changes I made (if applicable).
- [x] I added or updated documentation (if applicable).
- [x] I tried running the project locally and verified that there are no visible errors.

[best_practices]:https://git-scm.com/book/en/v2/Distributed-Git-Contributing-to-a-Project#_commit_guidelines

## Developer Certificate of Origin

<details>
<summary>Developer Certificate of Origin</summary>

```
Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
1 Letterman Drive
Suite D4700
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.


Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
```

</details>

### CODE REVIEW

File: src/components/VContentSwitcher/meta/VContentSwitcherPopover.stories.mdx [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+    select: {
+      action: 'select',
+    },
+  }}
+/>
+
+export const Template = (args, { argTypes }) => ({
+  template: `<VContentSwitcherPopover active-item="image" v-bind="$props" v-on="$props"/>`,
+  components: { VContentSwitcherPopover },
+  props: Object.keys(argTypes),
```
Comment (sarayourfriend) [inline src/components/VContentSwitcher/meta/VContentSwitcherPopover.stories.mdx:24]:
```suggestion
  template: `<VContentSwitcherPopover :active-item="activeItem" @select="setActiveItem" v-bind="$props" v-on="$props"/>`,
  components: { VContentSwitcherPopover },
  props: Object.keys(argTypes),
  data() {
    return {
      activeItem: 'image',
    }
  },
  methods: {
    setActiveItem(item) {
      this.activeItem = item
    }
  }
```

**Review by sarayourfriend — CHANGES_REQUESTED** [review]
Selecting an individual item doesn't work. Applying the suggested change should fix it 🙂 

I noticed a bug with the content switcher while testing this... `closeMenu` isn't used at all. I think there's a misunderstanding about how to use the `VPopover` component, as the `VPopoverContent` slot get's passed the `close` function that can be called when an item is selected. I'll open an issue to fix this.

**Comment by dhruvkb** [comment]
Thanks, I'll update it. My idea was to keep the story simple, allowing the `select` event to be sent to the Actions tab instead of handling it.

**Comment by sarayourfriend** [comment]
Ohhh, I wasn't even aware of what that actions tab did! Feel free to ignore my suggestion then.

**Review by sarayourfriend — APPROVED** [review]

**Comment by zackkrida** [comment]
> wasn't even aware of what that actions tab did

Same. TIL 😊

**Review by zackkrida — APPROVED** [review]
LGTM!

File: src/components/VContentSwitcher/meta/VContentSwitcherPopover.stories.mdx (hunk 1 above)
Comment (dhruvkb) [reply src/components/VContentSwitcher/meta/VContentSwitcherPopover.stories.mdx:24]: Skipping in favour of the Actions tab showing emitted events.

**Pull request merged**